frontend-instructor/src/components/courseManagement/QuizForm.jsx

```
@@ -0,0 +1,543 @@
+              className='px-4 py-2 bg-blue-600 text-white font-medium rounded-md hover:bg-blue-700 focus:outline-none focus:ring-2 focus:ring-blue-500 focus:ring-offset-2'
+              onClick={() => {
+                const questionElement = document.getElementById('new-question');
+                const pointsElement =
+                  document.getElementById('question-points');
+                const answerElement = document.getElementById('correct-answer');
+
+                const newQuestion = {
+                  question: questionElement.value,
+                  options: newOptions,
+                  answer: answerElement.value,
+                  points: parseInt(pointsElement.value, 10) || 5
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)